src/components/LyricView.tsx

```diff
@@ -59,29 +59,24 @@ const LyricView: React.FC<LyricViewProps> = (props: LyricViewProps) => {
     let key = 0;
 
     const songLyrics = song.lyrics;
-    const verseKeys = Object.keys(songLyrics);
-    const lyricBlocks: string[][] = Object.values(songLyrics);
+    const presentationOrder = song.presentation?.split(" ");
 
-    for (let i = 0; i < verseKeys.length; ++i) {
-      const verseKey = verseKeys[i];
-      const verseLyrics = lyricBlocks[i];
+    // if the song has a presentation order defined, use it;
+    // otherwise, present in written order
+    if (typeof presentationOrder !== "undefined") {
+      presentationOrder.forEach((verseNumber) => {
+        const verseName: string = getVerseText(verseNumber);
+        if (verseNumber === "c") {
+          verseNumber = "c1";
```

Comment:
If there are some presentation fields that have 'c' but should have 'c1', I would prefer to update them in the json rather than adjust them here.